proto/interchain_security/ccv/provider/v1/query.proto

```diff
@@ -342,10 +343,33 @@ message QueryConsumerValidatorsValidator {
   // The power of the validator used on the consumer chain
   int64 power = 3;
   // The rate to charge delegators on the consumer chain, as a fraction
-  string rate = 4 [
+  string consumer_commission_rate = 4 [
     (gogoproto.customtype) = "cosmossdk.io/math.LegacyDec",
     (gogoproto.nullable)   = false
     ];
+  // description defines the description terms for the validator
+  cosmos.staking.v1beta1.Description description = 5 [(gogoproto.nullable) = false];
+  // operator_address defines the address of the validator's operator
+  string operator_address = 6 [(cosmos_proto.scalar) = "cosmos.AddressString"];
```

Comment:
This is the operator address on the provider, right? If so, rename to `provider_operator_address`.

Reply:
Actually, given that multiple of these fields are part of the Validator struct, maybe just return that, i.e., 
```
cosmos.staking.v1beta1.Validator provider_validator
```

Reply:
The code does what was specified in the issue.

Operator address is not relevant for the consumer chain, so it's clear what it refers to.

Reply:
Renamed to `provider_operator_address`.